project-addons/custom_partner/partner_view.xml

```diff
@@ -404,6 +404,7 @@
             <field name="arch" type="xml">
                 <field name="lang" position="after">
                     <field name="growth_rate"/>
+                    <label for="average_margin"/><div><field name="average_margin" class="oe_inline"/> %% </div>
```

Comment:
¿Estáis seguro de que no necesitáis ocultar el campo por grupos? ¿probasteis a entrar con un perfil de comercial pelado o con un comercial externo?